.github/workflows/escalation-slack-notification.yml

```diff
@@ -0,0 +1,31 @@
+name: Escalation Slack Notification
+
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  on-escalation-label:
+    if: github.event.label.name == 'escalation'
```

Comment:
Although it seems that GitHub doesn't care about the label case, let's be explicit and make it match [the actual label](https://github.com/metabase/metabase/labels/Escalation).

Also @NevRA please commit this with `[ci skip]`.
It's pointless for the whole CI to run because of this workflow that cannot affect backend, E2E and frontend in any way.

Reply:
Done